packages/itwin/grouping-mapping-widget/src/widget/components/Grouping.tsx

```diff
@@ -242,12 +250,14 @@ export const Groupings = ({ mapping, goBack }: GroupsTreeProps) => {
     [groups, hiddenGroupsIds, getHiliteIdsFromGroups],
   );
 
-  const addGroup = () => {
+  const addGroup = (type: string) => {
```

Comment:
Why does this not use the `useCallback` hook?

Reply:
there is no heavy dependency